src/common/ClientHelper.ts

```diff
@@ -241,7 +241,7 @@ export async function constructClients(
     // ProposeRootBundle in order to match a bundle block evaluation block range with a pending root bundle.
     maxBlockLookBack: config.maxBlockLookBack[config.hubPoolChainId],
   };
-  const hubPoolClient = new HubPoolClient(logger, hubPool, config.hubPoolChainId, hubPoolClientSearchSettings);
+  const hubPoolClient = new HubPoolClient(logger, hubPool, 0, config.hubPoolChainId, hubPoolClientSearchSettings);
```

Comment:
Here's the eventual change that we want :+1:

```suggestion
  const hubPoolClient = new HubPoolClient(
    logger,
    hubPool,
    Number(getDeploymentBlockNumber("HubPool", config.hubPoolChainId)),
    config.hubPoolChainId,
    hubPoolClientSearchSettings
  );
```

Reply:
Updated in: https://github.com/across-protocol/relayer-v2/pull/662/commits/b3e1538a564521abbff11801d9f3c9e4e3e74c17